### ISSUE TITLE
Use texture cache maintenance utillity to clean thumbnails

### ISSUE THREAD

**Issue by tstrul1** [cibboy/script.profilecleaner#1]
http://kodi.wiki/view/Texture_Cache_Maintenance_utility
Officialy supported by kodi & clleaning operation is better with this tool.

**Comment by cibboy** [comment]
I may understand it wrong, but I think that that specific utility is standalone and not integrated into an addon that can be run from inside Kodi. This would probably complicate its usage in Profile Cleaner. Any ideas?

**Comment by tstrul1** [comment]
Yes this utility is standalone. I think it would be best if it would be integrated in an addon.
I remember that the developer of this utility said it is standalone because he is not sure if kodis python interpreter has all the modules needed but he didnt check it. I think it would be nice of someone will check it. and if it is so complicated maybe put a portable python folder inside the addon folder and just execute the script with it, it is a dirty solution but thats what i can think of... (I am not a coder :) 
(I usually run the tool with these flags vclean,P,Xd,R,C)
